src/swarm-lang/Swarm/Language/Direction.hs

```diff
@@ -50,7 +50,7 @@ data AbsoluteDir = DEast | DNorth | DWest | DSouth
   deriving (Eq, Ord, Show, Read, Generic, Data, Hashable, Enum, Bounded)
 
 directionJsonModifier :: String -> String
-directionJsonModifier = map C.toLower . L.tail
+directionJsonModifier = map C.toLower . L.drop 1
```

Comment:
Nice workaround for partiality

Reply:
Yeah, that's an easy one.  There's nothing like `drop` to replace `head`, but https://github.com/haskell/core-libraries-committee/blob/main/guides/warning-for-head-and-tail.md had some good ideas, including the idea to replace uses of `head` with a locally-defined version that pattern-matches and calls `error` explicitly.  That avoids the warning and also generates a better error message in case it ever does actually crash.